package/network/utils/umbim/files/lib/netifd/proto/mbim.sh

```diff
@@ -169,6 +169,8 @@ _proto_mbim_setup() {
 
 	echo "mbim[$$]" "Connected"
 
+	local zone="$(fw3 -q network "$interface" 2>/dev/null)"
```

Comment:
Does this still work with fw4?

Reply:
The same implementation is still used in other protocol handlers, and `/sbin/fw3` is just a symlink to to `/sbin/fw4`. Should I replace it? The same syntax with `fw4` works just as well as with `fw3`.

Reply:
ok thanks for confirming